<launch>
  <node pkg="mono_vo_ros" type="mono_vo_node" name="mono_vo_node"  output="screen"> 
    <param name="max_frame" value="4000" />
    <param name="min_num_feat" value="4000" />
    <param name="fn_kitti" value="$(env HOME)/dataset/kitti/data_odometry" />
  </node>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find mono_vo_ros)/rviz/mono_vo.rviz" output="screen" />
</launch>

